packages/syft/src/syft/core/node/common/node_service/testing_services/smpc_executor_service.py

```diff
@@ -0,0 +1,74 @@
+# stdlib
+from typing import List
+from typing import Type
+
+# third party
+from nacl.signing import VerifyKey
+
+# relative
+from ...... import lib
+from .....store.storeable_object import StorableObject
+from ....abstract.node import AbstractNode
+from ....common.action.smpc_action_message import SMPCActionMessage
+from ....common.action.smpc_action_message import _MAP_ACTION_TO_FUNCTION
+from ....common.node_service.node_service import ImmediateNodeServiceWithoutReply
+
+
+class SMPCExecutorService(ImmediateNodeServiceWithoutReply):
+    @staticmethod
+    def message_handler_types() -> List[Type[SMPCActionMessage]]:
+        return [SMPCActionMessage]
+
+    @staticmethod
+    def process(
```

Comment:
Can we get some comments and documentation on this service explaining what's going on? It's pretty complex :)

Reply:
Sure! I will have in the morning a sync with @madhavajay only to be sure that I might not overcomplicate stuff with the `celeryworker` and the MPCTensor and tomorrow I will get this done.

Reply:
brilliant <3

Reply:
Is this resolved?